nuxeo-features/rest-api/nuxeo-rest-api-server/src/main/java/org/nuxeo/ecm/restapi/server/jaxrs/JSONDocumentObject.java

```diff
@@ -56,15 +58,30 @@
 
 @WebObject(type = "Document")
 @Produces({ "application/json+nxentity", "application/json+esentity", MediaType.APPLICATION_JSON })
-public class JSONDocumentObject extends DocumentObject {
+public class JSONDocumentObject extends DefaultObject {
```

Comment:
Is `org.nuxeo.ecm.core.rest.DocumentObject` deprecated? Why this breaking API change?